Libraries/Components/View/ViewAccessibility.js

```diff
@@ -55,3 +57,16 @@ export type AccessibilityRole =
 
 // This must be kept in sync with the AccessibilityStatesMask in RCTViewManager.m
 export type AccessibilityStates = $ReadOnlyArray<'disabled' | 'selected'>;
+
+// the info associated with an accessibility action
+export type AccessibilityActionInfo = $ReadOnly<{
+  name: string,
+  label?: string,
+}>;
+
+// The info included in the event sent to onAccessibilityAction
+export type AccessibilityActionEvent = SyntheticEvent<
+  $ReadOnly<{
+    actionName: string,
```

Comment:
it would be great to align the "actionName" here with the "name" in `AccessibilityActionInfo`, so that it's clear these are the same thing.